Configuration/Geometry/python/GeometryRecoDB_cff.py

```diff
@@ -32,3 +32,6 @@ def _loadGeometryESProducers( theProcess ) :
 
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 modifyGeometryConfiguration_cff_ = run2_GEM_2017.makeProcessModifier( _loadGeometryESProducers )
+
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+modifyGeometryConfiguration_cff_ = run3_GEM.makeProcessModifier( _loadGeometryESProducers )
```

Comment:
@ianna each modifier object needs to have a unique name. I changed the names to `modifyGeometryConfigurationRun2_cff_` and `modifyGeometryConfigurationRun3_cff_`, and workflow 10024.0 step2 runs to completion.

Reply:
@kpedro88 - Thanks! I'll open a separate PR for it.